sdk-lib/src/main/assets/saplingtree/testnet/1600000.json

```diff
@@ -1,7 +1,7 @@
 {
   "network": "test",
   "height": "1600000",
-  "hash": "005ac505cbc14c9a32398af56a8af78fb8410e2a91a709bb713aa949461e40e3",
-  "time": 1633550806,
-  "tree": "01d6294d3a3a467279ec51ce982f74f1170d5dd7592802fc8de9ab647fdfb16a0f01b9d90d75dd22574755247d2a67527a097b115ea918404c9203b2e9e327c9642610011c5fe419180b55b88b59c1a27d7b6af280ce9b380acb5de50f7b40ffd867766d0000010d8d9e94a966082ab8752f08b218eb4a6537c2eab6b4ad639159e16874d11a2b00000000000117bce82d9d9f3fd58b4edfb4df6852d3322f451a80bf64351ef9fe7e51e1e21001bab80e68a5c63460d1e5c94ef540940792fa4703fa488b09fdfded97f8ec8a3d00013d2fd009bf8a22d68f720eac19c411c99014ed9c5f85d5942e15d1fc039e28680001f08f39275112dd8905b854170b7f247cf2df18454d4fa94e6e4f9320cca05f24011f8322ef806eb2430dc4a7a41c1b344bea5be946efc7b4349c1c9edb14ff9d39"
+  "hash": "001b64e46b60917ad763efd76967eac15507aa9161a0351325588bc748a0464d",
```

Comment:
@pacu, why do some of these testnet checkpoints have different hash, tree, and time values but still have the same height?

Reply:
@str4d can you take a look at this. Does it have anything to do with the rollback we did?

@ccjernigan which server did you use to generate the testnet checkpoints?

Reply:
great catch @ccjernigan

Reply:
@pacu I used testnet.lightwalletd.com:9067

Reply:
I checked both servers and are on the same branch. so these testnet are valid, they just have different values because of the chain rewind.

````
pacu@pacu ~ % grpcurl  testnet.lightwalletd.com:9067 cash.z.wallet.sdk.rpc.CompactTxStreamer/GetLightdInfo

{
  "version": "v0.4.10",
  "vendor": "ECC LightWalletD",
  "taddrSupport": true,
  "chainName": "test",
  "saplingActivationHeight": "280000",
  "consensusBranchId": "c2d6d0b4",
  "blockHeight": "1864415",
  "gitCommit": "68789356fb1a75f62735a529b38389ef08ea7582",
  "buildDate": "2022-04-30",
  "buildUser": "lightwalletd",
  "estimatedHeight": "1864415",
  "zcashdBuild": "v4.7.0-gitian",
  "zcashdSubversion": "/MagicBean:4.7.0/"
}
pacu@pacu ~ % grpcurl  lightwalletd.testnet.electriccoin.co:9067 cash.z.wallet.sdk.rpc.CompactTxStreamer/GetLightdInfo  

{
  "version": "v0.4.10",
  "vendor": "ECC LightWalletD",
  "taddrSupport": true,
  "chainName": "test",
  "saplingActivationHeight": "280000",
  "consensusBranchId": "c2d6d0b4",
  "blockHeight": "1864415",
  "gitCommit": "68789356fb1a75f62735a529b38389ef08ea7582",
  "buildDate": "2022-04-26",
  "buildUser": "root",
  "estimatedHeight": "1864415",
  "zcashdBuild": "v4.7.0-gitian",
  "zcashdSubversion": "/MagicBean:4.7.0/"
}
````